opennsa/config.py

```diff
@@ -350,18 +366,19 @@ def _load_certificates(self, vc):
         # tls
         if vc[TLS]:
             try:
-                hostkey = cfg.get(BLOCK_SERVICE, KEY)
-                hostcert = cfg.get(BLOCK_SERVICE, CERTIFICATE)
-
-                if not os.path.exists(hostkey):
+                if not vc[KEY]:
                     raise ConfigurationError(
-                        'Specified hostkey does not exist (%s)' % hostkey)
-                if not os.path.exists(hostcert):
+                        'must specify a key when TLS is enabled')
+                elif not os.path.exists(vc[KEY]):
                     raise ConfigurationError(
-                        'Specified hostcert does not exist (%s)' % hostcert)
```

Comment:
just a nicer pattern to read, nice addition in py 3.x I think.

```py
f'Specified hostcert does not exist ({hostcert})' 
```

Reply:
I agree that it is a nicer pattern. I only just followed the old pattern already used everywhere else in this file. I could change it but should I then not change all old string format patterns in this file (or maybe even the complete code base)?

 And it then probable should read `f'Specified certificate does not exist ({vc[CERTIFICATE]})`.

Reply:
I didn't mean to have you change the entire code base or file.  I tend to try to update things as I touch them.  So if I'm working on a piece of code I try to bring in better patterns.  It's just a nice to have either ways.

Reply:
It is still a good suggestion, but for now I would like not cloud this change with this improvement.